docs/14-batch-shipyard-configuration-jobs.md

```diff
@@ -1032,6 +1037,25 @@ property are:
           Azure Blob Storage URL.
         * `file_mode` if the file mode to set for the file on the compute node.
           This is optional.
+    * `pre_execution_command` is a command that is run only on the master node
+      of this multi-instance task prior to the application command. This
+      command must not block and must exit successfully for the multi-instance
+      task to proceed. This command can be used to populate environement
+      variables required to run the application command. This is optional and
+      may be null. For Docker containers, usually this property will be
```

Comment:
Remove this sentence. Make a logic change in convoy/settings to detect if this is set for a docker task and disallow it.